dtbase/models/arima.py

```diff
@@ -0,0 +1,550 @@
+import logging
+import os
+import sys
+from copy import deepcopy
+from datetime import timedelta
+from typing import Optional, Tuple, Union
+
+import coloredlogs
+import numpy as np
+import pandas as pd
+import pydantic
+from sklearn.metrics import mean_absolute_percentage_error, mean_squared_error
+from sklearn.model_selection import TimeSeriesSplit
+from statsmodels.tsa.statespace.sarimax import SARIMAX, SARIMAXResultsWrapper
+
+from dtbase.models.utils.config import read_config
+from dtbase.models.utils.dataprocessor.clean_data import clean_data, clean_data_list
+from dtbase.models.utils.dataprocessor.config import (
+    ConfigData,
+    ConfigOthers,
+    ConfigSensors,
+)
+from dtbase.models.utils.dataprocessor.get_data import get_training_data
+from dtbase.models.utils.dataprocessor.prepare_data import prepare_data
+from dtbase.services.base import BaseModel
+
+# Read default params from config file
+_config_file_path = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)), "config_arima.ini"
+)
+_defaults = read_config(_config_file_path, "arima")
+
+# set up logging
+logger = logging.getLogger(__name__)
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
```

Comment:
Does this line change the logging config globally whenever someone imports this module? I always forget how this logging config stuff works. If yes, that could perhaps be changed. In general, I think logging config setting is best left for scripts rather than library-like modules.

Reply:
Agreed

Reply:
I've actually removed all this formatting of the logs here. I think its probabaly uneccessary